apps/api/src/modules/summary/queues/summary.queue.ts

```diff
@@ -0,0 +1,5 @@
+export const SUMMARY_QUEUE = 'summary';
+
+export const summaryQueueConfig = {
+  name: SUMMARY_QUEUE,
+};
```

Comment:
Did you run the command to format from package.json?
Linting and formatting should be run before committing/pushing

Reply:
Not sure about this, i didn't make changes here or run any commands

Reply:
Run the format and lint command.
There are many issues in formatting.
Let me also check why its not triggered in CI.

Reply:
Added the lines in package.json and then ran the lint and format commands